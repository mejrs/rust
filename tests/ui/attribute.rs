#![crate_type = "lib"]

#[blah(x(y(note = concat!())))]
struct Foo;