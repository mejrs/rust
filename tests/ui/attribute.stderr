error: cannot find attribute `blah` in this scope
  --> $DIR/attribute.rs:3:3
   |
LL | #[blah(x(y(note = concat!())))]
   |   ^^^^

error: aborting due to 1 previous error

